### PR TITLE
Wix/os changelog

### PR DESCRIPTION
**Pull request by wicked-wix**
Jag har lagt till en subkategori under OpenStack IaaS för Changelogs där jag även la till changeloggen för vår kommande uppgradering (queens to train)

### CODE REVIEW

File: content/en/docs/openstack-iaas/changelog/train.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Changelog for OpenStack Train"
+description: "Upgrade of version Queens to Train"
+alwaysopen: true
+---
+
+#### TL;DR
```
Comment (Brittlejf) [inline content/en/docs/openstack-iaas/changelog/train.md:7]:
```suggestion
#### Changelog overview
```

Comment (Brittlejf) [reply content/en/docs/openstack-iaas/changelog/train.md:7]: I think we should change the TL;DR heading to something more professional.

File: content/en/docs/openstack-iaas/changelog/train.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+#### Visible changes (Horizon/web UI)
+* The “Overview” page will be divided into categories by resource type. It will also contain information about more resources.
+* “Server Groups” are now visible under the “Compute” heading.
+* “Consistency Groups” and “Consistency Groups Snapshots” have been replaced by “Groups” and “Group Snapshots” under the “Volume” heading.
+* It is now possible to manage “Application Credentials” under the “Identity” heading.
+#### New features
+The upgrades come with a lot of new features, such as:
+* UDP load balancers in Octavia (Train)
+* [Fine grained access rules](https://docs.openstack.org/api-ref/identity/v3/#application-credentials "Application Credentials") can now be defined for “Application Credentials” (Train)
+##### Reference
```
Comment (Doctor-love) [inline content/en/docs/openstack-iaas/changelog/train.md:27]: In these cases, we are starting at heading 5 and go down to heading 5 for "Reference". Consider starting at heading 1 ("#") or 2 ("##") instead?

Comment (Brittlejf) [reply content/en/docs/openstack-iaas/changelog/train.md:27]: It's probably a good idea to leave a linebreak between the headings and paragraphs as well for formatting purposes.

Comment (wicked-wix) [reply content/en/docs/openstack-iaas/changelog/train.md]: Have fixed both now

**Review by MrMoreau — APPROVED** [review]
LGTM

**Pull request merged**